### ISSUE TITLE
Inconsistency between the .txt file and the identifiability tableau plot

### ISSUE THREAD

**Issue by X-M-YUAN** [genssi-developer/GenSSI#25]
I tried to run a structural identifiability analysis on a model with two parameters kds and ds, and the .txt file shows they are both structurally globally identifiable:

![Capture](https://github.com/user-attachments/assets/8e9b8156-2f80-4122-84b6-b6e0ba26a97b)

However, the identifiability tableau plot is very weird because it is completely empty, which means that they are structurally non-identifiable:

![Capture](https://github.com/user-attachments/assets/53cab6df-e758-4455-9216-c4a929a3c022)

What is the issue here?

**Comment by thomassligon** [comment]
Could you send me the model (via email)? Then I could try running it.

Tom
(Dr. Thomas S. Ligon)
***@***.******@***.***>
Frohnloher Str. 6a
81475 Muenchen
Germany
Tel. +49(89)74575075

From: X-M-YUAN ***@***.***>
Sent: Wednesday, November 6, 2024 4:54 PM
To: genssi-developer/GenSSI ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [genssi-developer/GenSSI] Inconsistency between the .txt file and the identifiability tableau plot (Issue #25)


I tried to run a structural identifiability analysis on a model with two parameters kds and ds, and the .txt file shows they are both structurally globally identifiable:

Capture.PNG (view on web)<https://github.com/user-attachments/assets/8e9b8156-2f80-4122-84b6-b6e0ba26a97b>

However, the identifiability tableau plot is very weird because it is completely empty, which means that they are structurally non-identifiable:

Capture.PNG (view on web)<https://github.com/user-attachments/assets/53cab6df-e758-4455-9216-c4a929a3c022>

What is the issue here?

**Comment by X-M-YUAN** [comment]
Hi Thomas

Please find in the attachment to get the model.

Regards

Xuming Yuan

**Comment by genssi-developer** [comment]
I don't see an attachment with the model, and I don't understand what is happening.

**Comment by X-M-YUAN** [comment]
Hello

I am sending the models to you again. Can you find them in the attachment this time?

Regards

Xuming Yuan

**Comment by thomassligon** [comment]
No, I don’t see an attachment. In addition, I find it difficult to read your HTML-formatted email, and my Outlook does not recognize that is should be reading HTML instead of normal text.

Maybe the attachment was suppressed by ma email-server, due to this warning:
** THIS MESSAGE ORIGINATED OUTSIDE LOUGHBOROUGH UNIVERSITY ** <br> <br> ** Be wary of links or attachments, especially if the email is unsolicited or you don&#39;t recognise the sender&#39;s email address
However, I didn’t see the attachment in GitHub, so this problem seems to have occurred earlier.

Tom
(Dr. Thomas S. Ligon)
***@***.******@***.***>
Frohnloher Str. 6a
81475 Muenchen
Germany
Tel. +49(89)74575075
